l10n_fi_reconcile/static/src/xml/account_reconciliation.xml

```diff
@@ -7,21 +7,28 @@
             <td>
                 <t t-if="line.ref" t-esc="line.ref"/>
             </td>
+            <td>
+                <t t-if="line.partner_note" t-esc="line.partner_note" />
```

Comment:
This line still needs a clarifying comment that points out that it is not actually a field value, but a spoof.